### ISSUE TITLE
Provide a means of filtering errors

### ISSUE THREAD

**Issue by kloots** [rackt/react-a11y#34]
If you use react-a11y on an existing project, you'll likely be overwhelmed by the number of accessibility errors. It'd be useful to have a means of filtering errors by component name, so developers can easily focus on addressing failures on a per-component basis.

**Comment by AsaAyers** [comment]
It's not possible to discover the component names from inside `react-a11y`. See https://github.com/rackt/react-a11y/issues/4#issuecomment-77160916

**Comment by AsaAyers** [comment]
oh, are you talking about component name being `<div>` and not `<MyCustomComponent>`?

**Comment by kloots** [comment]
Here's the approach I came up with:

https://github.com/rackt/react-a11y/pull/36

**Comment by AsaAyers** [comment]
Sorry, I didn't notice who made the issue and I replied without reading the other notifications :(

Great job finding a solution! I looked at the code a bit and it sounds great. Much better than what I came up with :)
